packages/babel-preset-gatsby/src/__tests__/index.js

```diff
@@ -0,0 +1,19 @@
+const babel = require(`babel-core`)
+const preset = require(`../`)
+
+function matchesSnapshot(query, options = {}) {
+  const { code } = babel.transform(query, {
+    presets: [[preset, options]],
+  })
+  expect(code).toMatchSnapshot()
+}
+
+it(`Applies @babel/preset-env for target browsers`, () => {
```

Comment:
This is more of like an integration test. I'd maybe just test the actual contents of the preset (i.e. correct presets/packages/etc). but this is a cool test!

Reply:
Makes sense. I needed to sanity check my `presets` option. I can work on this at the end 🙂